packages/puppeteer-core/src/common/bidi/BrowsingContext.ts

```diff
@@ -289,6 +294,7 @@ export class BrowsingContext extends EventEmitter {
       context: this.#id,
     });
     const sessionId = session.result.cdpSession;
+    this.#cdpSessionId = sessionId;
```

Comment:
Should we skip the first call if we have a `cdpSessionId` set?

Reply:
I am not sure about the lifetime of sessions on the mapper side? perhaps we could